podman/domain/networks_manager.py

```diff
@@ -161,8 +161,8 @@ def list(self, **kwargs) -> List[Network]:
         return [self.prepare_model(i) for i in response.json()]
 
     def prune(
-        self, filters: Optional[Dict[str, Any]] = None
-    ) -> Dict[api.Literal["NetworksDeleted", "SpaceReclaimed"], Any]:
+        self, filters: Optional[dict[str, Any]] = None
+    ) -> dict[api.Literal["NetworksDeleted", "SpaceReclaimed"], Any]:
```

Comment:
```suggestion
    ) -> dict[Literal["NetworksDeleted", "SpaceReclaimed"], Any]:
```
Do we need to use api.Literal any more?

Reply:
probably not, I'll handle it in a separate commit